src/components/victory-line/victory-line.js

```diff
@@ -40,12 +43,39 @@ export default class VictoryLine extends React.Component {
   static defaultTransitions = {
     onExit: {
       duration: 500,
-      before: (datum) => ({ y: datum.y })
+      before: (datum) => ({ y: datum.y }),
+      beforeClipPathWidth: (data, child, exitingNodes) => {
```

Comment:
It looks like they are slightly different because your two `getScale` methods return differently. I think the static methods should return the same thing in all the components. I think in VictoryArea this might mean calculating the domain twice, but I think it's worth it for the code cleanliness. And I think it can be refactored so that doesn't happen with a little change to `helpers/scale` I know I'm very picky in this regard, and I am happy to jump in on refactoring that part if you want to move on.